security/application.md

```diff
@@ -0,0 +1,555 @@
+encryption, asymmetric encryption, and hashing functions to transmit data
+securely and efficiently.
+
+It does _not_ guarantee that the domain owner is trustworthy. TLS does not
+relate to trust. It can only guarantee that the information is sent, untampered
+and privately, only to the recipient you are sending it to. It does not
+guarantee that you are sending it to the right recipient.
+
+There are two kinds of certificates signed by certificate authorities: domain
+verification and extended verification. Domain verification does what it says
+on the tin: it confirms that the holder of the private key is also in control
+of the domain name. Extended verification goes further and cannot be automated:
+it confirms that the holder of the private key controls the domain name and is
+the person or company they claim to be.
+
+Extended verification does not guarantee that the holder of the private key is
+trustworthy.
+
+We mostly related to TLS via HTTPS, but it can be used for any TCP connection,
+such as email.
+
+The first version of TLS was named Secure Sockets Layer (SSL); at the end of
+the last century, SSL was found to be trivially vulnerable and has not been
+intentionally used since.
+
+### HSTS
+
+The most common attack vector for a secure protocol is a downgrade attack. Many
+protocols have a backward compatibility mechanism that allows the client and
+server to negotiate which version of a protocol they both understand.
+Convincing the server to downgrade to a version of a protocol with a known bug
+is a downgrade attack.
+
+The worst case is when you can convince the server to drop the security
+entirely. This is possible with HTTPS with a standard man-in-the-middle attack
+of the _unencrypted_ HTTP connection.
+
+It works like this:
+
+1. User visits `thoughtbot.com`.
+2. Web browser turns this into `http://thoughtbot.com/`.
+3. An eavesdropper intercepts the connection and redirects to their own server,
+   secured using TLS, but entirely under their control.
+
+The problem is step (2). The current solution is HTTP Strict Transport Security
+(HSTS). Under HSTS, the browser knows about domain names that should always be
+HTTPS. It literally has a list. If you enter `thoughtbot.com` into the URL bar,
+it will check its list, find `thoughtbot.com` in there, and complete the full
+URL as `https://thoughtbot.com/`.
+
+Web sites can add themselves to the list by sending the
+`Strict-Transport-Security` header. The value for this header is
+`max-age=31536000; includeSubDomains; preload`.
+
+- `max-age` determines how long this domain should remain in the browser's
+  list. `31536000` is one year. Feel free to go longer.
+- `includeSubDomains` specifies that subdomains should also be considered part
+  of the HSTS list.
+- `preload` tells the browser maintainer that you are comfortable with this
+  domain being part of [the list shipped with the
+  browser](https://hstspreload.org/).
```

Comment:
hstspreload.org says that thoughtbot.com isn't part of [the list](https://hstspreload.org/?domain=thoughtbot.com)? Am I misunderstanding how the site works?